src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -49,107 +39,51 @@ export const App = ({ loggedIn, toggleLoginModal, displayToggle }) => {
       });
   };
 
-  const getLastUpdatedOn = () => {
+  const fetchAvailableForms = () => {
     return apiRequest('/form1095_bs/available_forms')
       .then(response => {
         if (response.errors || !response.availableForms.length) {
           updateFormError({ error: true, type: 'not found' });
         }
-        return response.availableForms[0];
+        return response.availableForms;
       })
       .catch(() => updateFormError({ error: true, type: 'not found' }));
   };
 
-  const callLastUpdated = () => {
-    getLastUpdatedOn().then(result => {
-      if (result.lastUpdated && result.year) {
-        const date = new Date(result.lastUpdated);
-        const options = { year: 'numeric', month: 'long', day: 'numeric' };
-        // expected output (varies according to local timezone and default locale): December 20, 2012
-        updateLastUpdated(date.toLocaleDateString(undefined, options));
-        updateYear(result.year);
-      } else {
-        updateFormError({ error: true, type: 'not found' });
-      }
-    });
-  };
-
-  const callGetContent = () => {
-    getContent().then(result => {
+  const downloadFileToUser = format => {
+    fetchFile(format).then(result => {
       if (result) {
         const a = document.createElement('a');
         a.href = result;
         a.target = '_blank';
-
-        if (formType === 'txt') a.download = `1095B-${year}.txt`; // download text file directly
+        a.download = `1095B-${year}.${format}`;
 
         document.body.appendChild(a); // we need to append the element to the dom -> otherwise it will not work in firefox
         a.click();
         a.remove(); // removes element from the DOM
-        const date = new Date();
         updateFormError({ error: false, type: '' });
-        updateFormDownloaded({
-          downloaded: true,
-          timeStamp: formatTimeString(
-            date.toLocaleDateString(undefined, dateOptions),
-          ),
-        });
       }
     });
   };
 
   useEffect(
     () => {
-      callLastUpdated();
```

Comment:
The linter initially told me to add `callLastUpdated` to the watched array along with loggedIn on line 84 below. But once I did that, it said the method should be defined within the useEffect.